src/test/resources/features/Space.feature

```diff
@@ -1,7 +1,9 @@
 Feature: Create new space
 
+  @logout
   Scenario: The user create a new space
     Given The user goes to "login" page
-     And The "user" fills the form with email and password
-    When The user create a new space with the following name test1
-    Then The space name with the name test1 appear in the panel successfully
\ No newline at end of file
+    And The "user" fills the form with email and password
+    When The user create a new space with the following name test7
+    Then The space name with the name test7 appear in the panel successfully
+    
```

Comment:
Remove this space in white and only put a salt of line.

Reply:
Ok, I did it.